src/test/sock_tests.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2021-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef WIN32
```

Comment:
Q: Why are these tests disabled for Windows?

Reply:
They were because Windows does not have `socketpair()`, but since not all tests use this function I have now enabled the tests that don't.